src/components/ZeroLogon/Zerologon.tsx

```diff
@@ -1,46 +1,64 @@
-    "Step 5: Click Exploit to commence ZeroLogon’s operation.\n\n" +
-    "Step 6: View the Output block below to view the results of the attack vectors execution.";
-
+/**
+ * Represents the form values for the ZeroLogon component.
+ */
 interface FormValues {
     dcName: string;
     targetIP: string;
     adminName: string;
     hashes: string;
 }
 
-export function ZeroLogon() {
-    const [loading, setLoading] = useState(false);
-    const [output, setOutput] = useState("");
-    const [pid, setPid] = useState(""); // Maintain the state of the process id.
-    const [allowSave, setAllowSave] = useState(false);
-    const [hasSaved, setHasSaved] = useState(false);
+/**
+ * The ZeroLogon component.
+ * @returns The ZeroLogon component.
+ */
+const ZeroLogon = () => {
+    // Component State Variables.
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state.
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+    const [allowSave, setAllowSave] = useState(false); // State variable to allow saving the output.
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if the output has been saved.
+    const [opened, setOpened] = useState(true); // State variable to indicate if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+
+    // Component Constants.
+    const title = "ZeroLogon"; // Title of the component.
+    const description =
+        "The ZeroLogon CVE allows an attacker that has unauthenticated access to a domain controller within their network " +
+        "access to create a Netlogon session that can be exploited to grant domain administrative privileges. The vulnerability " +
+        "here lays within an implementation flaw for AES-CFB8 where a cryptographic transformation takes place with use of a " +
+        "session key.\n\nFurther information can be found at: https://www.crowdstrike.com/blog/cve-2020-1472-zerologon-" +
+        "security-advisory/\n\n" +
+        "Using ZeroLogon:\n\n" +
+        "Step 1: Enter a Domain Controller name.\n" +
+        "       Eg: TEST-AD\n\n" +
+        "Step 2: Enter a Target IP address.\n" +
+        "       Eg: 192.168.1.1\n\n" +
+        "Step 3: Enter an Administrative name @ Domain Controller IP.\n" +
+        "       Eg: Administrator\n\n" +
+        "Step 4: Enter any relevant Hashes.\n" +
+        "       Eg: Administrator:500:CEEB0FA9F240C200417EAF40CFAC29C3:D280553F0103F2E643406517296E7582:::\n\n" +
+        "Step 5: Click Exploit to commence ZeroLogon’s operation.\n\n" +
+        "Step 6: View the Output block below to view the results of the attack vectors execution.";
```

Comment:
Needs an apostrophe. 
```suggestion
        "Step 6: View the Output block below to view the results of the attack vector's execution.";
```